docs/server/source/production-deployment-template/architecture.rst

```diff
@@ -0,0 +1,70 @@
+Architecture of an IPDB Node
```

Comment:
nit: may be change the name to IPDB reference architecture just IPDB architecture? what does the node refer to here?

Reply:
`Node` refers to a `BigchainDB Node in a Federation`.
I would suggest that the title remains the same as the entire docs are in the `production deployment template` and can be customized by users in whatever way they seems to be fit.